config/routes.rb

```diff
@@ -449,6 +449,8 @@
     end
   end
 
+  resources :courses
```

Comment:
To make sure that Rails handles the fact that we're not using all the builtin routes we should add 

```
only: [:show, :index]
```